### PR TITLE
Add kyverno deployment to app root

### PR DESCRIPTION
**Pull request by screeley44**
Adding in deployment, SA, Services, policy and config maps to the app root for kyverno

### CODE REVIEW

**Comment by screeley44** [comment]
@HumairAK @harshad16  - how does this look for app root components?

File: kyverno/overlays/moc/smaug/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../../base
```
Comment (harshad16) [inline kyverno/overlays/moc/smaug/kustomization.yaml:6]: need an empty line at the end.

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
a minor addition needed

File: kyverno/overlays/moc/smaug/kustomization.yaml (hunk 1 above)
Comment (screeley44) [reply kyverno/overlays/moc/smaug/kustomization.yaml]: fixed thanks

**Comment by codificat** [comment]
/close
as we are going with `cosigned` instead (#1560)

**Pull request closed without merging**